src/routes.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Route, IndexRoute } from 'react-router';
+
+import {App} from './components/app';
+import {Login} from './components/pages/login';
+import {AdminRoutes} from './components/pages/admin';
+import {TrainingPage, StudentsRoutes} from './components/pages/students';
+import {NoMatchPage} from './components/pages/noMatch';
+
+const componentTrainers = () : any => {
+  // https://medium.com/@Nadav.Dav/implementing-lazy-loading-with-webpack-and-react-router-f8497f895892#.mq4ug0c15
+  return {
+      getComponent: (location, cb) => {
+        require.ensure(['./components/pages/trainers'], require => {
+              // Retrieve product page component
+              cb(null, require('./components/pages/trainers')["default"]);
+        });
+      }
+   };
+};
+
+
+export const AppRoutes = (
+    <Route  path="/" component={App} >
+      <IndexRoute component={Login}/>
+      <Route path="/home" component={Login} />
+      <Route path="/trainers" {...componentTrainers}/>
+      {AdminRoutes}
+      {StudentsRoutes}
+      <Route path="*" component={NoMatchPage} />
+    </Route>
```

Comment:
It would be great to follow same patter for all the groups, just to let it prepared for future implementation.

I would expect something like:

```
export const AppRoutes = (
     <Route  path="/" component={App} >
       {CommonRoutes}
       {TrainersRoutes}
       {AdminRoutes}
       {StudentsRoutes}
    </Route>
```

Then in each subfolder just add routes and a simple page if there is only one (something silly like hello from trainers Module).